README.md

```diff
@@ -111,11 +111,11 @@ Use the provided snapshots to quickly sync your Juno node with the current state
 | ------- | ---- | ----- | ------------- |
 | **>=v0.9.2**  | **156 GB** | **519634** | [**juno_mainnet.tar**](https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.9.3_519634.tar) |
 
-#### Goerli
+#### Sepolia
 
 | Version | Size | Block | Download Link |
 | ------- | ---- | ----- | ------------- |
-| **>=v0.9.2** | **48 GB** | **931113** | [**juno_goerli.tar**](https://juno-snapshots.nethermind.dev/goerli/juno_goerli_v0.9.2_931113.tar) |
```

Comment:
Do we need to remove this file as well in S3 ?

Reply:
It's not s3, it's r2 cloud, but yeah we should remove. 
cc: @Manjeet-Nethermind